### DISCUSSION TITLE
Unreachable-Fehlermeldung per Checkbox abschaltbar?

### DISCUSSION THREAD

**Discussion by fraenki57** [MK-2001/ioBroker.go-e#133] [Ideas]
Moin,
wäre es machbar, die Fehlermeldung, dass der go-e nicht (via angegebener IP-Adresse) erreichbar ist, abgeschaltet werden kann (z.B. via Checkbox in der Adapter-Konfig)
Grund bei mir ist, dass ich zwei go-e habe, die aber vergleichsweise selten brauche, und diese daher erst bei Bedarf per VIS einschalte, um die 2 x 4W StandBy-Stromverbrauch einzusparen (das summiert ja ja doch ganz schön über die Zeit).
Im abgeschalteten Zustand wird dann das Protokoll mit entsprechenden Logs beider go-e ("connect EHOSTUNREACH 192.168.9.xx:80") überschwemmt, so dass andere "echte" Fehlermeldungen anderer Adapter untergehen.
Danke!

**Comment by MK-2001** [comment]
Hi,

Wäre es dann nicht besser den Adapter per Script zu deaktivieren?

Ich kann es aber gerne mit aufnehmen. Zumindest, dass man das LogLevel für diese Meldung einstellen kann. Per Set auf Info sollte es dann weg sein.

Wäre das auch okay?

Gruß

**Reply by fraenki57** [reply]
Moin,

klar ginge es auch, den Adapter per Skript zu deaktivieren, muss man aber erst mal per Skript manuell einrichten ... und ich wüsste jetzt z.B. nicht, wie man einen Adapter per Skript deaktiviert ... müsste ich erst mühsam(?) ausforschen.
Das über die Konfig zu machen, finde ich eleganter.

>> Zumindest, dass man das LogLevel für diese Meldung einstellen kann. Per Set auf Info sollte es dann weg sein. Wäre das auch okay?

Ja, ich denke, das ist auch eine Lösung, die das Ziel meines Vorschlags erfüllt. :-)

Danke für dein Engagement!

Gruß Frank
